documentation/source/base/components.rst

```diff
@@ -64,3 +65,34 @@ for instance the separatrix of a plasma.
             )
 
 A :py:class:`ComponentManager` should be how a :py:class:`Component` is used after creation within the top level of the reactor design.
+
+Reactor
+-------
+
+:py:class:`Reactors` are again designed to be created by the :ref:`Reactor Designer <how to use>`.
+This object is the complete reactor and is a container that allows easy access to any part of it.
+Methods on the :py:class:`Reactor` object have access to all parts of the reactor
+enabling functionality that need to interact with multiple :py:class:`ComponentManagers`.
+
+.. code-block:: python
+
+    class MyReactor(Reactor):
+        '''An example of how to declare a reactor structure.'''
+
+        plasma: MyPlasma
+        tf_coils: MyTfCoils
+
+        def get_ripple(self):
+            '''Calculate the ripple in the TF coils.'''
+
+    reactor = MyReactor("My Reactor")
+    reactor.plasma = build_plasma()
+    reactor.tf_coils = build_tf_coils()
+    reactor.show_cad()
+
+A :py:class:`Reactor` interacts dynamically with :py:class:`ComponentManagers`.
```

Comment:
Only a small thing (and may just be my personal preference), but I would usually put the `s` outside the directive, since the class name is actually `ComponentManager` (note you'll need to escape the `s` - `` :py:class:`ComponentManager`\s ``).

```suggestion
A :py:class:`Reactor` interacts dynamically with :py:class:`ComponentManager`\s.
```

If you can be bothered, you could even do a find-and-replace on these to make them `` :py:class:`~bluemira.base.reactor.ComponentManager` `` - then they'll link to the API docs.

Reply:
Done!